SUMO-254678: Doc update Database apps migration V1 to V2 (#5009)
SumoLogic/sumologic-documentation: docs/integrations/databases/redis.md

This section provides instructions for configuring log and metric collection for the Sumo Logic app for Redis. Configuring log and metric collection for the Redis ULM app includes the following tasks:

### Step 1: Configure fields in Sumo Logic

> As part of the app installation process, the following fields will created by default:

Reviewer: ```suggestion
As part of the app installation process, the following fields will be created by default:
```

* `component`
* `environment`
* `db_system`
* `db_cluster`
* `db_cluster_address`
* `db_cluster_port`

> Additionally, if you are using Redis in the Kubernetes environment, these following additional fields will be created by default as a part of the app installation process:

Reviewer: ```suggestion
Additionally, if you are using Redis in the Kubernetes environment, the following additional fields will be created by default during the app installation process:
```

* `pod_labels_db_cluster_port`

> To learn more about setting up fields, refer to [Sumo Logic Fields](/docs/manage/fields).

Reviewer: ```suggestion
For information on setting up fields, see [Fields](/docs/manage/fields).
```

    kubectl describe pod <redis_pod_name>
    ```
   5. Sumo Logic Kubernetes collection will automatically start collecting logs from the pods having the annotations defined above.

> <br/>**FER to normalize the fields in Kubernetes environments.** Labels created in Kubernetes environments automatically are prefixed with pod_labels. To normalize these for our app to work, we will have Field Extraction Rule automatically created for Database Application Components named as **AppObservabilityRedisDatabaseFER**.

Reviewer: ```suggestion
<br/>**FER to normalize the fields in Kubernetes environments.** Labels created in Kubernetes environments automatically are prefixed with pod_labels. To normalize these for our app to work, a Field Extraction Rule named **AppObservabilityRedisDatabaseFER** is automatically created for Database Application Components.
```
